pages/common/acme.sh.md

```diff
@@ -0,0 +1,36 @@
+# acme.sh
+
+> An ACME Shell script, an acme client alternative to certbot.
+> More information: <https://github.com/acmesh-official/acme.sh>.
+
+- Issue a certificate using webroot mode:
+
+`acme.sh --issue --domain {{example.com}} --webroot {{/path/to/webroot}}`
+
+- Issue a certificate using standalone mode using port 80:
+
+`acme.sh --issue --standalone --domain {{example.com}}`
+
+- Issue a certificate using standalone TLS mode using port 443:
+
+`acme.sh --issue --alpn --domain {{example.com}}`
+
+- Issue a certificate using a working Nginx configuration:
+
+`acme.sh --issue --nginx --domain {{example.com}}`
+
+- Issue a certificate using a working Apache configuration:
+
+`acme.sh --issue --apache --domain {{example.com}}`
+
+- Issue a wildcard (\*) certificate using a manual DNS mode:
+
+`acme.sh --issue --dns --domain {{example.com}}`
+
+- Issue a certificate using an automatic DNS API mode:
+
+`acme.sh --issue --dns {{dns_api}} --domain {{example.com}}`
```

Comment:
Perhaps a concrete example for `dns_api` would aid clarity here?

Reply:
@sbrl sure.

Instead of `{{dns_api}}` would be one of the many available options specified here https://github.com/acmesh-official/acme.sh/wiki/dnsapi

So, for instance for CloudFlare DNS API it is `{{dns_cf}}`

Also, in example used `{{dns_api}}` is still perfectly valid for custom scripts, if there is a `dns_api.sh` script present in the `.acme.sh/` root folder, more info https://github.com/acmesh-official/acme.sh/issues/1571#issuecomment-384999814

Reply:
Maybe this would also deserved it's separate subcommand page?

Reply:
But is it actually a subcommand?  If so, then if should just be `dns_api.sh.md` (if we document it, which we probably will).

Reply:
I believe it would be called more like `acme.sh-dns.md`, as the subcommand in question is named `--dns {{ }}`

Reply:
If we document it as a subcommand, we'd probably do something similar to pacman in #5268, for example. It sounds like the DNS option here is more complex than I thought, so let's leave this particular example as-is.

Reply:
@sbrl it is already open in https://github.com/tldr-pages/tldr/pull/5852#issuecomment-830626806